pkg/api/controller_test.go

```diff
@@ -26,7 +26,11 @@ func testHTTPResponse(t *testing.T, r *gin.Engine, req *http.Request, f func(w *
 }
 
 func TestGetServers(t *testing.T) {
-	myConfigFile := config.InitConfig()
+	myConfigFile, err := config.InitConfig()
+	if err != nil {
+		t.Errorf("Error from InitConfig come normally from a broken configFile. TODO : Test env. must be sandboxed. Then config must be mocked.")
```

Comment:
`An error from InitConfig usually comes from a broken configFile. TODO : Test env. must be sandboxed. Then config must be mocked.`

Reply:
`An error from InitConfig usually comes from a broken configFile. TODO : Test env. must be sandboxed. Then config must be mocked.`